bodhi/server/bugs.py

```diff
@@ -176,10 +176,10 @@ def on_qa(self, bug_id: int, comment: str):
         try:
             bug = self.bz.getbug(bug_id)
             if bug.product not in config.get('bz_products'):
-                log.info("Skipping set on_qa on {0!r} bug #{1}".format(bug.product, bug_id))
+                log.info("Skipping set on_qa on %s bug #%s", bug.product, bug_id)
```

Comment:
You may want to have `repr(bug.product)` here to replace the `{!r}`.

Reply:
@abompard have updated @bowlofeggs commit with the suggested changes.